packages/nextjs/src/utils/refreshAccessTokenIfNeeded/index.ts

```diff
@@ -32,9 +33,11 @@ export default async function refreshAccessToken(ctx: NextPageContext): Promise<
   }
 
   try {
-    logger.info(`Check if should request made from first application load`);
+    logger.info(
+      `Check if should request made from first application load, disableInitialPropsRefreshToken: ${config.disableInitialPropsRefreshToken}`
+    );
 
-    if (isRuntimeNextRequest(url)) {
+    if (isRuntimeNextRequest(url) || config.disableInitialPropsRefreshToken) {
```

Comment:
Breaking change, because getEnv every user who is not passing `DISABLE_INITIAL_PROPS_REFRESH_TOKEN` will get an error here

Reply:
change to getEnvOrDefault